CMake/cdat_modules/vacumm_deps.cmake

```diff
@@ -0,0 +1 @@
+set(vacumm_deps ${python_pkg} ${numpy_pkg} ${scipy_pkg} ${matplotlib_pkg} ${basemap_pkg} ${configobj_pkg})
```

Comment:
Seawater isn't a hard dependency - only for advanced diags: http://relay.actimar.fr/~raynaud/vacumm/user.install.prereq.html